data/test/scenarios/test_for_tag.cfg

```diff
@@ -56,6 +92,15 @@
     [/event]
 )}
 
+#####
+# API(s) being tested: [for]
+##
+# Actions:
+# Use the [for] tag with only end=-2.
+##
+# Expected end state:
+# The contents of the loop is not executed.
```

Comment:
Is this a bug? According to the docs, this should be equivalent to `start=0` and then since `start > end` it should imply `step = -1` and execute 3 times.

Reply:
Perhaps it's meant to be "when start > end it should imply step = -1, **when end is positive**"? I'm not sure why that would be the case, but my assumption is this is the intended functionality given this is what the test is confirming to happen.